dali/python/nvidia/dali/pipeline.py

```diff
@@ -846,6 +846,11 @@ def feed_input(self, data_node, data, layout=None, cuda_stream=None, use_copy_ke
         In the case of the GPU input, the data must be modified on the same stream as the one
         used by ``feed_input``. See ``cuda_stream`` parameter for details.
 
+        In the case of using asynchronous executor mode the DALI executor can consume
+        `prefetch_queue_depth` and 1 more if the input is consumed directly by the mixed operator.
+        This is needed as the CPU operators can have up to `prefetch_queue_depth` buffers
+        precomputed, mixed operator will have one more in transit.
+
```

Comment:
Honestly, I don't understand this paragraph - and that's saying a lot given that I'm rather familiar with our execution model.

Reply:
What kind of wording would you suggest then?

Reply:
...also, I'm not sure that the part about mixed operators is actually true.

Reply:
```suggestion
    In order to avoid stalls in asynchronous pipelines, the data should be provided ahead of time
    `prefetch_queue_depth` times.
```

Reply:
Done